docs/source/api_reference.rst

```diff
@@ -68,6 +68,12 @@ Expression Navigator
         :members:
         :undoc-members:
 
+DatasetsUtil
+------------
+.. automodule:: genestack_client.datasets_util
```

Comment:
`DatasetsUtil` has only one class, wouldn't it be better to document it as `autoclass`, like `FilesUtil`?